.github/workflows/cmake.yml

```diff
@@ -15,6 +15,7 @@ jobs:
             cmake_generator: "Unix Makefiles"
             cmake_options:
               -DPA_USE_OSS=ON
+              -DPA_USE_SNDIO=ON
```

Comment:
I don't think this should be necessary considering CMake should default to enabling sndio if the required library is found.